addContact.php

```diff
@@ -26,8 +29,8 @@
             $email = $_POST['email'];
             $relationship = $_POST['relationship'];
 
-            $stmt = $db->prepare("INSERT INTO Contact (Student_Id, First_Name, Last_Name, Phone_Cell, Phone_Home, Address, City, State, Zip, Email, Relationship) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
-            $stmt->bind_param('isssssssiss', $id, $fname, $lname, $cellphone, $homephone, $address, $city, $state, $zip, $email, $relationship);
+            $stmt = $db->prepare("INSERT INTO Contact (Student_Id, Prefix, First_Name, Middle_Name,Last_Name, Suffix ,Phone_Cell, Phone_Home, Address, City, State, Zip, Email, Relationship) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
+            $stmt->bind_param('isssssssiss', $id,  $prefix, $fname, $mname ,$lname, $suffix,$cellphone, $homephone, $address, $city, $state, $zip, $email, $relationship, $references);
```

Comment:
we are going to want to remove the relationship field from here, we will hold that information in the student_to_contacts table